packages/http/src/mocker/negotiator/types.ts

```diff
@@ -1,12 +1,13 @@
 import { IHttpOperationConfig } from '@stoplight/prism-http';
-import { IHttpHeaderParam, INodeExample, INodeExternalExample, ISchema } from '@stoplight/types';
+import { IHttpHeaderParam, INodeExample, INodeExternalExample } from '@stoplight/types';
+import { JSONSchema4, JSONSchema6, JSONSchema7 } from 'json-schema';
 
 export interface IHttpNegotiationResult {
   code: string;
   mediaType: string;
   bodyExample?: INodeExample | INodeExternalExample;
-  schema?: ISchema;
-  headers: IHttpHeaderParam[];
+  schema?: JSONSchema4 | JSONSchema6 | JSONSchema7;
```

Comment:
Have you thought about creating a `type JSONSchema = JSONSchema4 | JSONSchema6 | JSONSchema7` and exporting it in `@stoplight/types`?

Reply:
Yes, 2-3 times, but at the end I didn't do it because hopefully we'll come up with the right schema draft we want to support and we'll probably remove all the other.

I understand it's kind of a big overload though, so I can do the change. In another PR :trollface:

Reply:
Cool!